packages/next/src/server/config.ts

```diff
@@ -45,11 +45,6 @@ const experimentalWarning = execOnce(
       `Experimental features are not covered by semver, and may cause unexpected or broken application behavior. ` +
         `Use at your own risk.`
     )
-    if (features.includes('appDir')) {
-      Log.info(
-        `Thank you for testing \`appDir\` please leave your feedback at https://nextjs.link/app-feedback`
-      )
-    }
```

Comment:
Perhaps we should warn here when `appDir: true` to inform the user they no longer need config 😄

Reply:
No relevant for right now, it can be added later 👍